bin/qc.py

```diff
@@ -226,45 +254,52 @@ def go(args):
     # Pangolin Lineages
     lineage = get_lineage(args.pangolin, args.sample)
 
-    # ncov-tools
-    ncov_tools_status = get_ncovtools_qc(args.ncovtools, args.sample).replace(',', '|')
+    # NCOV-Tools Results
+    summary_df = parse_ncov_tsv(args.ncov_summary, args.sample)
+    negative_df = parse_ncov_tsv(args.ncov_negative, args.sample, negative=True)
 
     if args.sample_sheet:
         run_name, barcode, project_id, ct = get_samplesheet_info(args.sample_sheet, args.sample)
     
     else:
-        run_name = 'N/A'
+        run_name = 'NA'
         barcode = re.search(r'\d+', args.sample).group(0)
-        project_id = 'N/A' 
-        ct = 'N/A'
-
-
-    qc_line = { 'sample_name' : args.sample,
-                 'project_id' : project_id,
-                    'barcode' : barcode,
-                    'count_N' : count_N,
-                'pct_N_bases' : "{:.2f}".format(pct_N_bases),
-          'pct_covered_bases' : "{:.2f}".format(pct_covered_bases), 
-           'longest_no_N_run' : largest_N_gap,
-             'depth_coverage' : depth_coverage,
-          'num_aligned_reads' : num_reads,
-                    'lineage' : lineage,
-                   'variants' : variants,
-'diagnostic_primer_mutations' : primer_statement,
-                         'ct' : ct,
-                   'run_name' : run_name,
-                'script_name' : 'nml-ncov2019-artic-nf',
-                   'revision' : args.revision,
-              'ncov-tools-qc' : ncov_tools_status,
-                    'qc_pass' : qc_pass}
-
-
-    with open(args.outfile, 'w') as csvfile:
-        header = qc_line.keys()
-        writer = csv.DictWriter(csvfile, fieldnames=header)
-        writer.writeheader()
-        writer.writerow(qc_line)
+        project_id = 'NA' 
+        ct = 'NA'
+
+
+    qc_line = {      'sample' : [args.sample],
+                 'project_id' : [project_id],
+                    'barcode' : [barcode],
+                    'lineage' : [lineage],
+                   'variants' : [variants],
+'diagnostic_primer_mutations' : [primer_statement],
+             'run_identifier' : [run_name],
+                'script_name' : ['nml-ncov2019-artic-nf'],
+                   'revision' : [args.revision]}
+
+    qc_df = pd.DataFrame.from_dict(qc_line)
+
+    ## Old ##
+    # with open(args.outfile, 'w') as csvfile:
```

Comment:
get rid of commented code. If you need to ref back, use git.

Reply:
removed